docker-compose.yml

```diff
@@ -79,6 +77,55 @@ services:
         source: $DDS_LOG_DIR
         target: /dds_web/logs
 
-#  redis:
-#    container_name: dds_redis
-#    image: redis:latest
+  minio:
+    container_name: dds_minio
+    image: minio/minio:RELEASE.2022-02-24T22-12-01Z
+    profiles:
+      - s3
+      - full-dev
+      - cli
+    command: server /data --console-address ":9001"
+    ports:
+      - 127.0.0.1:9000:9000
+      - 127.0.0.1:9001:9001
+    environment:
+      MINIO_ROOT_USER: minio  # access key
+      MINIO_ROOT_PASSWORD: minioPassword  # secret key
+    # volumes:
+    #   - type: bind
+    #     source: ./minio-data
+    #     target: /data
```

Comment:
Just checking if you intend to keep this or if you just forgot to remove it?

Reply:
I thought to keep in case people want to keep their data. Have been thinking of adding a similar thing to the db as well. Will remove it if people disagree with me.

Reply:
It's nice, but maybe needs a comment explaining why it's here then.. https://github.com/ScilifelabDataCentre/dds_web/pull/993/files#r819352572